package.json

```diff
@@ -38,18 +38,22 @@
     "@commitlint/config-angular": "8.2.0",
     "@commitlint/prompt": "8.2.0",
     "@commitlint/prompt-cli": "8.2.0",
-    "@edx/frontend-build": "^2.0.4",
-    "@edx/frontend-platform": "1.1.6",
-    "@edx/paragon": "7.1.5",
+    "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
+    "@edx/frontend-build": "5.4.0",
+    "@edx/frontend-platform": "1.8.0",
+    "@edx/paragon": "12.0.5",
     "codecov": "3.7.2",
     "enzyme": "3.10.0",
     "enzyme-adapter-react-16": "1.14.0",
     "husky": "3.0.9",
     "prop-types": "15.7.2",
     "react": "16.10.2",
     "react-dom": "16.10.2",
+    "react-redux": "^7.1.1",
```

Comment:
react-redux, react-router-dom, and redux were added as dev deps to satisfy the example app which was refactored to use `AppProvider` from `edx/frontend-platform` such that the config settings can be pulled from `AppContext`.